camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -335,7 +335,7 @@ def mk_object_points(self, boards, use_board_size = False):
             num_pts = b.n_cols * b.n_rows
             opts_loc = numpy.zeros((num_pts, 1, 3), numpy.float32)
             for j in range(num_pts):
-                opts_loc[j, 0, 0] = (j / b.n_cols)
+                opts_loc[j, 0, 0] = (j // b.n_cols)
```

Comment:
Double slash?

Reply:
Makes sure it returns an int which was the default behavior in Py2 for a division (py3 will return a float, https://stackoverflow.com/questions/1535596/what-is-the-reason-for-having-in-python)

Reply:
Well I think that shows that it returns a floored float, ei `2.000` not `2`.

Reply:
In any case it was going to be converted to float when assigned to the array, but the result of the right hand expression is now the same as the previous version with Python 2.